toolkit/tools/pkg/imagecustomizerlib/grubcfgutils.go

```diff
@@ -590,25 +590,38 @@ func GrubArgsToString(args []string) string {
 	return combinedString
 }
 
-// Converts an SELinux mode into the list of required command-line args for that mode.
-func selinuxModeToArgs(selinuxMode imagecustomizerapi.SELinuxMode) ([]string, error) {
-	newSELinuxArgs := []string(nil)
+// Helper function to get common SELinux args based on mode
+func getCommonSELinuxArgs(selinuxMode imagecustomizerapi.SELinuxMode) ([]string, error) {
 	switch selinuxMode {
 	case imagecustomizerapi.SELinuxModeDisabled:
-		newSELinuxArgs = []string{installutils.CmdlineSELinuxDisabledArg}
-
+		return []string{installutils.CmdlineSELinuxDisabledArg}, nil
 	case imagecustomizerapi.SELinuxModeForceEnforcing:
-		newSELinuxArgs = []string{installutils.CmdlineSELinuxSecurityArg, installutils.CmdlineSELinuxEnabledArg,
-			installutils.CmdlineSELinuxEnforcingArg}
-
+		return []string{installutils.CmdlineSELinuxSecurityArg, installutils.CmdlineSELinuxEnabledArg,
+			installutils.CmdlineSELinuxEnforcingArg}, nil
 	case imagecustomizerapi.SELinuxModePermissive, imagecustomizerapi.SELinuxModeEnforcing:
```

Comment:
Would it make sense to add a new SELinux mode called `SELinuxModeForcePermissive`, which is the same as `SELinuxModePermissive` but adds `enforcing=0`?

Reply:
Sounds good. Can I make this a follow-up change so it can be included in EMU to get the Trident pipeline passing?

Reply:
Sure